app/models/event.rb

```diff
@@ -0,0 +1,15 @@
+class Event < ApplicationRecord
+  scope :past_events, -> { where('date < ?', Time.now)}
+  scope :upcoming_events, -> { where('date > ?', Time.now)}
```

Comment:
What about events being equal to Time.now? Are they upcoming or past? This is better being `>=`.

[**REQUIRED**]